packages/codegen-ui/lib/validation-helper.ts

```diff
@@ -173,7 +177,7 @@ const studioThemeValuesSchema: any = yup.object({
 });
 
 const studioThemeSchema = yup.object({
-  name: pascalCaseAlphaNumString().required(),
+  name: alphaNumString().required(),
```

Comment:
maybe a better name is camelCaseAlphaNumString

Reply:
will fix as a follow up pr needs to ref this pr convo